rclcpp/include/rclcpp/node.hpp

```diff
@@ -232,13 +232,15 @@ class Node : public std::enable_shared_from_this<Node>
    * \param[in] period Time interval between triggers of the callback.
    * \param[in] callback User-defined callback function.
    * \param[in] group Callback group to execute this timer's callback in.
+   * \param[in] amount_of_callbacks Quantity of times the callback will be triggered.
    */
   template<typename DurationRepT = int64_t, typename DurationT = std::milli, typename CallbackT>
   typename rclcpp::WallTimer<CallbackT>::SharedPtr
   create_wall_timer(
     std::chrono::duration<DurationRepT, DurationT> period,
     CallbackT callback,
-    rclcpp::CallbackGroup::SharedPtr group = nullptr);
+    rclcpp::CallbackGroup::SharedPtr group = nullptr,
+    unsigned int amount_of_callbacks = 0);
```

Comment:
this variable is of type `uint32_t` in all the other instances, please use the same type also here for consistency

Reply:
Sure thing. Done.